docs/appendix/glossary/subtype.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Subtype
+id: subtype
+full_link:
+short_description: A particular implementation of a resource type. For example, an arm is a subtype of the component resource type.
+aka:
+---
+
+A particular implementation of a {{< glossary_tooltip term_id="type" text="type" >}} of {{< glossary_tooltip term_id="resource" text="resource" >}}, as delineated by its {{< glossary_tooltip term_id="api-namespace-triplet" text="api-namespace-triplet" >}}
```

Comment:
```suggestion
A category within a {{< glossary_tooltip term_id="type" text="type" >}} of {{< glossary_tooltip term_id="resource" text="resource" >}}.
All resources of a given subtype share the same API.
Models are subcategories within that subtype, implementing the common API.
```
"as delineated by its api-namespace-triplet" makes this first sentence hard to parse. Let's introduce them to the concept in a more concrete way before getting into that more technical piece (see suggestion below for where to put it).

Reply:
I agree less technical but to keep some of that phrasing in mind for people maybe 
```suggestion
A category within a {{< glossary_tooltip term_id="type" text="type" >}} of {{< glossary_tooltip term_id="resource" text="resource" >}}.
Resources belonging to a subtype share the same API.
{{< glossary_tooltip term_id="model" text="Models" >}} implement that subtype's API protocol.
``` 
?

Reply:
Okay sure! 
```suggestion

A category within a {{< glossary_tooltip term_id="type" text="type" >}} of {{< glossary_tooltip term_id="resource" text="resource" >}}.
Resource models belonging to a subtype share the same API.
{{< glossary_tooltip term_id="model" text="Models" >}} implement that subtype's API protocol with different drivers.
```
I think adding "models" to the second line will help too.